src/methods/relative_partial_randomization.jl

```diff
@@ -0,0 +1,89 @@
+export RelativePartialRandomization, RelativePartialRandomizationAAFT
+
+"""
+    RelativePartialRandomization(α = 0.5)
+`RelativePartialRandomization` surrogates are similar to [`PartialRandomization`](@ref)
+phase surrogates, but instead of drawing phases uniformly from `[0, 2π]`, phases are drawn
+from `ϕ + [0, 2π]*α`, where `α ∈ [0, 1]` and `ϕ` is the original Fourier phase.
+
+See the documentation for a detailed comparison between partial randomization algorithms.
+"""
+struct RelativePartialRandomization{T} <: Surrogate
+    α::T
+
+    function RelativePartialRandomization(α::T=0.5) where T <: Real
+        @assert 0 <= α <= 1
```

Comment:
Don't use `@assert`. It can be disabled globally. Use formal error checking: `0 <= α <= 1 || throw(ArgumentError("α must be between 0 and 1").`